packages/dd-trace/test/config.spec.js

```diff
@@ -1479,7 +1479,7 @@ describe('Config', () => {
     const config = new Config()
 
     config.configure({
-      trace_sample_rules: [
+      tracing_sampling_rules: [
```

Comment:
Is this a semver major change?

Reply:
no, it is only receiving the key-value pair from backend. I changed the key so that it can receive what the backend sends